IntegrationTestSuite/WebservicesIntegrationTests/Tests/EngineeringModel/RuleVerification/RuleVerificationTestFixture.cs

```diff
@@ -104,39 +104,74 @@ public void VerifyThatARuleVerificationCanBeCreatedWithWebApi()
-            Assert.IsFalse((bool)builtInRuleVerification[PropertyNames.IsActive]);
+            Assert.AreEqual("4efbc475-37f3-4219-8571-e896a78545d5", (string) builtInRuleVerification[PropertyNames.Iid]);
+            Assert.AreEqual(2, (int) builtInRuleVerification[PropertyNames.RevisionNumber]);
+            Assert.AreEqual("BuiltInRuleVerification", (string) builtInRuleVerification[PropertyNames.ClassKind]);
+            Assert.AreEqual("Test Built In Rule Verification", (string) builtInRuleVerification[PropertyNames.Name]);
+            Assert.AreEqual("NONE", (string) builtInRuleVerification[PropertyNames.Status]);
+            Assert.IsNull((string) builtInRuleVerification[PropertyNames.ExecutedOn]);
+            Assert.IsFalse((bool) builtInRuleVerification[PropertyNames.IsActive]);
 
             // get a specific RuleVerification from the result by it's unique id
-            var userRuleVerification = jArray.Single(x => (string)x[PropertyNames.Iid] == "486f4b97-fcb1-409e-b5c0-057c240f41b6");
+            var userRuleVerification = jArray.Single(x => (string) x[PropertyNames.Iid] == "486f4b97-fcb1-409e-b5c0-057c240f41b6");
 
             // verify the amount of returned properties 
             Assert.AreEqual(7, userRuleVerification.Children().Count());
 
             // assert that the properties are what is expected
-            Assert.AreEqual("486f4b97-fcb1-409e-b5c0-057c240f41b6", (string)userRuleVerification[PropertyNames.Iid]);
-            Assert.AreEqual(2, (int)userRuleVerification[PropertyNames.RevisionNumber]);
-            Assert.AreEqual("UserRuleVerification", (string)userRuleVerification[PropertyNames.ClassKind]);
-            Assert.AreEqual("NONE", (string)userRuleVerification[PropertyNames.Status]);
-            Assert.IsNull((string)userRuleVerification[PropertyNames.ExecutedOn]);
-            Assert.IsFalse((bool)userRuleVerification[PropertyNames.IsActive]);
-            Assert.AreEqual("8a5cd66e-7313-4843-813f-37081ca81bb8", (string)userRuleVerification[PropertyNames.Rule]);
+            Assert.AreEqual("486f4b97-fcb1-409e-b5c0-057c240f41b6", (string) userRuleVerification[PropertyNames.Iid]);
+            Assert.AreEqual(2, (int) userRuleVerification[PropertyNames.RevisionNumber]);
+            Assert.AreEqual("UserRuleVerification", (string) userRuleVerification[PropertyNames.ClassKind]);
+            Assert.AreEqual("NONE", (string) userRuleVerification[PropertyNames.Status]);
+            Assert.IsNull((string) userRuleVerification[PropertyNames.ExecutedOn]);
+            Assert.IsFalse((bool) userRuleVerification[PropertyNames.IsActive]);
+            Assert.AreEqual("8a5cd66e-7313-4843-813f-37081ca81bb8", (string) userRuleVerification[PropertyNames.Rule]);
+        }
+
+        [Ignore("Doen't work because RuleVerification ordered implementation is not OK")]
+        public void VerifyThatARuleVerificationCanBeReorderedWithWebApi()
```

Comment:
i propose we add a GH ticket reference to fix this

Reply:
Was already done. Added the issue URL to the text in the Ignore Attribute.